menu.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
-
 <head>
 	<title>PizzaParty</title>
```

Comment:
this should probably say PizzaParty - Menu

Reply:
menu added